robotoff/app/api.py

```diff
@@ -1106,3 +1160,4 @@ def on_get(self, req: falcon.Request, resp: falcon.Response, username: str):
 api.add_route("/api/v1/health", HealthResource())
 api.add_route("/api/v1/dump", DumpResource())
 api.add_route("/api/v1/users/statistics/{username}", UserStatisticsResource())
+api.add_route("/api/v1/insights/predictions/", PredictionCollection())
```

Comment:
Oh I didn't notice the url, early on. I think `/api/v1/predictions`  would be better
```suggestion
api.add_route("/api/v1/predictions/", PredictionCollection())
```